### ISSUE TITLE
Can't read custom headers

### ISSUE THREAD

**Issue by Dabrox02** [postalsys/imapflow#229]
**Describe the bug**
I am trying to read custom headers using the headers attribute of the queryFetchObject however I am not getting any values.

**To Reproduce**
1. Create an imapFlow instance
2. Connect
3. Make a query using fetch
4. Set the headers attribute with the custom header in this case 'x-forwaded-to'
5. Print the messages

`    
const searchObject = {
      since: thirtyMinutesAgo,
      all: true,
    }

    const queryFetchObject = {
      uid: true,
      envelope: true,
      bodyStructure: true,
      headers: ['date', 'subject', 'x-forwaded-to'],
    }

    const recentMessages = client.fetch(searchObject, queryFetchObject)
    
        for await (const msg of recentMessages) {
      console.log('headers', msg.headers.toString('utf-8'))
    }
   `

**Expected behavior**
You should see that it only prints date, subject but not x-forwade-d

**Desktop (please complete the following information):**
 - OS: Linux Mint 5.15.0-119-generic 64 bits 
 - Browser [e.g. chrome, safari]: brave-browser --version

**Comment by andris9** [comment]
Does the email even include that header? If you set headers to true then ImapFlow returns all available headers from the email.

**Comment by Dabrox02** [comment]
> Does the email even include that header? If you set headers to true then ImapFlow returns all available headers from the email.

Yes, the email contains the X-FORWARDED-TO header. I tried adding the value headers = true however it does not show all the headers, only a few.
